tasks/ports/freetds.rb

```diff
@@ -23,7 +23,7 @@ def configure_defaults
       end
 
       if windows?
-        opts << '--sysconfdir=C:\Sites'
+        opts << '--sysconfdir=C:/Sites'
```

Comment:
Well, to clarify my understanding is that the system path separator on Windows is "\" but almost all APIs support implicit normalization to the system path separator. Common exceptions to this rule are situations where you're using a software set which may be using its own file handling and relying on windows path separators or explicitly disabling normalization in order access previously invalid paths which are legal in NTFS or paths which extend beyond the 259 character boundary. In most cases this doesn't matter as long as you always use the system path separator and not the alt separator.

That said in this case you're absolutely right, it should be fine and since it was a bug it's a good fix. I just wasn't sure how it would come out the other end after passing through msys2 as it's entirely new to me. :)

Thank you very much for the clarification. :+1: